css/style.css

```diff
@@ -381,3 +381,94 @@ em {
 }
 
 } /*closing the media*/
+
+
+@media print {
```

Comment:
Most of this stuff i just stole from the HTML5 Boilerplate project. Then I added stuff that's custom to your site towards the bottom